### DISCUSSION TITLE
Two Setups

### DISCUSSION THREAD

**Discussion by TorbenMei** [Koenkk/zigbee2mqtt#18572] [Devices]
Does anyone have experience running two setups of Zigbee2MQTT in parallel and is it even possible?
/Torben

**Comment by francisp2** [comment]
Just use another topic and another channel

**Reply by TorbenMei** [reply]
Can two Zigbee2MQTT setup then be running on the same server?

**Reply by francisp2** [reply]
in /opt, create a directory zigbee2mqtt for your first instance, and a zigbee2mqtt2 for your second. 
Create an /etc/systemd/system/zigbee2mqtt.service and an /etc/systemd/system/zigbee2mqtt2.service, each pointing to the correct directory.

**Reply by TorbenMei** [reply]
Thank you very much for your interest and response.
I use Zigbee2MQTT in connection with OpenHab (RPi4), it is through Openhabian I have installed Zigbee2MQTT, which is fairly automatic, so therefore your suggestion is not entirely obvious to me.
But if you think it's possible, I'll dive into it.
Do I also get two frontends then?

**Reply by TorbenMei** [reply]
Hi again
I have now made a test setup of OpenHab in windows (on a NUC) - Zigbee2MQTT is also running and I think I can now actually figure out how to make instance no. two of Zigbee2MQTT. I'm looking forward to trying - but now work is calling.
